niworkflows/viz/plots.py

```diff
@@ -37,13 +37,13 @@ def __init__(self, func_file, mask_file=None, data=None, conf_file=None, seg_fil
         func_nii = nb.load(func_file)
         self.tr = tr if tr is not None else func_nii.header.get_zooms()[-1]
 
-        self.mask_data = np.ones_like(func_nii.get_data(), dtype='uint8')
+        self.mask_data = np.ones_like(func_nii.get_fdata(), dtype='uint8')
         if mask_file:
-            self.mask_data = nb.load(mask_file).get_data().astype('uint8')
+            self.mask_data = nb.load(mask_file).get_fdata().astype('uint8')
```

Comment:
Man, this is tedious. Hopefully nipy/nibabel#844 will go in, and we can start doing `np.uint8(nb.load(mask_file).dataobj)`, post-3.0.

```suggestion
            self.mask_data = np.asanyarray(nb.load(mask_file).dataobj).astype('uint8')
```